content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -310,6 +310,36 @@ assuming that the number of replicas is not also changed).
 A ReplicaSet can be easily scaled up or down by simply updating the `.spec.replicas` field. The ReplicaSet controller
 ensures that a desired number of Pods with a matching label selector are available and operational.
 
+### Pod Deletion Cost 
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+Using the `controller.kubernetes.io/pod-deletion-cost` annotation, users can set a preference regarding 
+which pods to remove first when downscaling a ReplicaSet.
+
+The annotation should be set on the pod, and the expected type is `int32`. It represents the cost of
+deleting a pod compared to other pods belonging to the same ReplicaSet. Pods with lower deletion
+cost are preferred to be deleted before pods with higher deletion cost. 
+
+The implicit value for this annotation for pods that don't set it is 0; negative values are permitted.
+Invalid values will be rejected by the API server.
+
+This feature is alpha and disabled by default. You can enable it by setting the
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`PodDeletionCost` in both kube-apiserver and kube-controller-manager.
+
+{{< warning >}}
+- This is honored on a best-effort basis, so it does not offer any guarantees on pod deletion order.
+- Users should avoid updating the annotation frequently (e.g., updating it based on a metric value)
+  because doing so will generate a significant number of pod updates on the apiserver.
+{{< /warning >}}  
```

Comment:
A note is enough here. Warning callouts are for “you could get hacked if you misunderstand this / omit a step” levels of importance.

Aside: we also avoid writing “e.g.” and other abbreviations for Latin phrases.

Reply:
done.